parcelLab/parcelLab-js-plugin
# REQ-115/bring-V4-up-to-date-with-translations

**Description:**
V4 is a branch of a very old version of V3, which does not support all the languages and uses it outdated language codes.

Bringing this version up to date with the V3 is possible, but the number of unexpected changes that could appear to the UI, that the vendor might not be happy about, might be too many. So I updated just the necessary part.

**Changes:**
This PR updates the languages and its translations.

PS: I kept the country names out as this logic is not present in this version of the plugin and it could make the PR cleaner to review.

**How to test:**
You can run in your local the repo farfetch.delivery-status.com and the js plugin with this branch.

Make sure that the farfetch.delivery-status.com on your local looks instead for the local js / css files being served.

Test the urls
http://localhost:3000/src/?locale=jp-ja&orderNo=ZD8D5H&show_articleList=yes&selectedTrackingNo=9579297965&s=yeHJzLTnft&lang=ja

http://localhost:3000/src/?locale=ko-ko&orderNo=ZD8D5H&show_articleList=yes&selectedTrackingNo=9579297965&s=yeHJzLTnft&lang=ko
